[PATCH] mysql: make it be able to work

Upstream-Status: Pending

the mysql.err log files is installed under /var/log/, so mysqld and
mysqld_safe_t should have the permission to read this path.

if /var/lib is not created, the mysqld_safe will create it, to install
the pid and data

Signed-off-by: Roy Li <rongqing.li@windriver.com>
---
 policy/modules/contrib/mysql.te | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/policy/modules/contrib/mysql.te b/policy/modules/contrib/mysql.te
index a819fa5..e418b89 100644
--- a/policy/modules/contrib/mysql.te
+++ b/policy/modules/contrib/mysql.te
@@ -162,6 +162,12 @@ allow mysqld_safe_t self:capability { chown dac_override fowner kill };
 allow mysqld_safe_t self:process { setsched getsched setrlimit };
 allow mysqld_safe_t self:fifo_file rw_fifo_file_perms;
 
+
+
+logging_list_logs(mysqld_safe_t)
+logging_list_logs(mysqld_t)
+files_var_lib_filetrans(mysqld_safe_t, mysqld_db_t, {file dir})
+
 allow mysqld_safe_t mysqld_t:process signull;
 
 read_lnk_files_pattern(mysqld_safe_t, mysqld_db_t, mysqld_db_t)
-- 
1.9.1

